src/app/shared/cms/components/cms-product-list-filter/cms-product-list-filter.component.ts

```diff
@@ -0,0 +1,49 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { Observable } from 'rxjs';
+import { filter, map, switchMap, take } from 'rxjs/operators';
+
+import { ShoppingFacade } from 'ish-core/facades/shopping.facade';
+import { ContentPageletView } from 'ish-core/models/content-view/content-view.model';
+import { CMSComponent } from 'ish-shared/cms/models/cms-component/cms-component.model';
+
+@Component({
+  selector: 'ish-cms-product-list-filter',
+  templateUrl: './cms-product-list-filter.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CMSProductListFilterComponent implements CMSComponent, OnChanges {
+  @Input() pagelet: ContentPageletView;
+
+  productSKUs$: Observable<string[]>;
+
+  constructor(private shoppingFacade: ShoppingFacade) {}
+
+  ngOnChanges() {
+    if (this.pagelet.hasParam('Filter') && this.pagelet.hasParam('Scope')) {
+      this.productSKUs$ = this.getProductSKUs$();
+    }
+  }
+
+  getProductSKUs$(): Observable<string[]> {
+    return this.shoppingFacade.selectedCategoryId$.pipe(
+      filter(categoryId => !!categoryId || (!categoryId && this.pagelet.stringParam('Scope') === 'GlobalScope')),
```

Comment:
Lets talk about the usage of `!!`. Do you actually need a boolean value here?

Reply:
filter operator needs a boolean as return value. 

export declare function filter<T>(predicate: (value: T, index: number) => boolean, thisArg?: any): MonoTypeOperatorFunction<T>;

Reply:
Good to learn. ;) So this is totally fine here.